starter-code/iteration2/article-with-comments/src/app/components/comments/comments.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <div class="content">
+    <h1>Comments</h1>
+    <p>Joe Johnson</p>
+    <p>I think this is a really nice place!</p>
```

Comment:
This can be placed inside the `comments.component.ts`, so then we only have to reference where with the interpolation `{{ ... }}`